.licenserc.yaml

```diff
@@ -0,0 +1,38 @@
+header:
```

Comment:
Looks like you are sneaking some license changes with this PR. I would like it better if we separated unrelated things but it is fine to keep if you want to.

Reply:
This was added as part of this PR because the tests would otherwise fail due to the fact that the copyright year has changed on the headers, so I had to fix it to 2023.

Reply:
I guess on this you could create a separate PR for the copyright and then merge main back into this branch